monkey/infection_monkey/exploit/wmiexec.py

```diff
@@ -28,8 +29,13 @@ class WmiExploiter(HostExploiter):
     def _exploit_host(self) -> ExploiterResultData:
 
         creds = generate_brute_force_combinations(self.options["credentials"])
+        intp_creds = interruptable_iter(creds,
+                                        self.interrupt,
+                                        "WMI exploiter has been interrupted by a stop signal from the Island",
```

Comment:
WMI exploiter shouldn't assume the signal has come from the island. "Signal" also has a special meaning (e.g. TERM, KILL, etc.) which could be confusing.

```suggestion
                                        "WMI exploiter has been interrupted",
```